packages/core/src/executors/publish/schema.d.ts

```diff
@@ -0,0 +1,8 @@
+import { dotnetPublishFlags } from '@nx-dotnet/dotnet';
+
+export type PublishExecutorSchema = {
+  [key in dotnetPublishFlags]?: string | boolean;
+} & {
+  publishProfile?: string;
+  extraParameters?: string;
```

Comment:
Should add typings here for the other properties (framework, force, output etc), even though we are not using those explicitly. These files might be used by something like jsdoc / tsdoc in near future for documentation.

Reply:
Do you mean in addition to the mapped type, or in replacement of the mapped type? Those properties are already included through the `[key in dotnetPublishFlags]` mappings, and they are visible in intellisense, though they all have `string | boolean` as their type, rather than their exact type (`string` for framework, `boolean` for force, etc.).

Reply:
Ah, gotcha. That does make sense, only downside is that tsdoc will not work with it well. Thats fine for now, whenever I get back to documentation I will re-evaluate.